test/integration/index.spec.ts

```diff
@@ -1426,75 +1425,6 @@ describe('Worker', () => {
 		await expect(ctx.flush(ctx.session)).rejects.toThrowError();
 	});
 
-	it('should not re-enqueue requests after execute failure', async () => {
```

Comment:
This test is a bit complex to read, but I think is testing a condition that we should test on the new implementation. Or at least describe what's the behavior. Adding some comments below

Reply:
I removed it because I think it becomes a duplicate of https://github.com/product-os/jellyfish-worker/pull/1836/files#diff-22706bc5498cd36bebba5565f3bc4dbe567effc30b01841bf096bfce9a2bb2a3R341 after the refactor. The idea is that the worker shouldn't attempt to execute the same action-request more than once regardless of if it passed or failed. Now that I think about it, I'm not sure if that conflicts with your recent changes of adding retries?